source/constructs/api/catalog/service.py

```diff
@@ -1234,6 +1234,26 @@ def fill_catalog_labels(catalogs):
     return catalogs
 
 
+def rebuild_catalog_labels(catalogs):
+    result = []
+    for catalogDic in catalogs:
+        catalog = catalogDic.CatalogDatabaseLevelClassification
+        catalog.labels = []
+        if catalog.database_type == DatabaseType.S3_UNSTRUCTURED.value:
+            catalog.table_name = catalog.storage_location
+        if catalog is not None and catalog.label_ids is not None and len(catalog.label_ids) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)